Samples/C#/CalculatorTest/AdvancedScenarios.cs

```diff
@@ -17,79 +17,51 @@
 using System;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using OpenQA.Selenium.Remote;
+using OpenQA.Selenium.Appium.Windows;
 
 namespace CalculatorTest
 {
     [TestClass]
-    public class AdvancedScenarios
+    public class AdvancedScenarios : CalculatorTestBase
```

Comment:
It gets awkward if we do that.  The basicscenario tests then become part of the advancedscenario test group.